### PR TITLE
0.4.0 - a batch of little tweaks and fixes

### PR DESCRIPTION
**Pull request by koreno**

### CODE REVIEW

File: easypy/collections.py [hunk 1]
```diff
@@ -553,27 +560,28 @@ def index(self, obj):
...
-        uid = self._get_object_uid(obj)
-        if uid not in self._objects:
-            raise ObjectNotFound(self, (), dict(key=uid),)
-        prev_uid = self._objects._OrderedDict__map[uid].prev
-        try:
-            key = prev_uid.key
-        except AttributeError:
-            key = prev_uid.prev.key
-        return self._objects[key]
+    if SUPPORT_GET_SIBLING:
```
Comment (idanarye) [inline easypy/collections.py]: We should have an `else:` with O(n) implementations, just to avoid this weird breakage.

Comment (koreno) [reply easypy/collections.py]: will do, but I'm actually thinking to remove get_prev/get_next, I don't think they ever became very popular...

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+	- the env-var `TERM_COLOR_SUPPORT` is no longer checked
+	- the env-var `EASYPY_LOG_INDENTATION` is no longer checked
+- DataSize - return 'byte' and 'MiB' instead of '1 byte' and '1 MiB'
+
+### Added
+- ziplog
+  - expose as a cli tool - `ziplog`
+  - added commonly found timestamp patterns
+- colors
+  - expose as a cli tool - `colorize
```
Comment (DoronCohen) [inline CHANGELOG.md]: close backtick

**Comment by idanarye** [comment]
@DoronCohen I still have some unanswered comments...

**Comment by DoronCohen** [comment]
@koreno are you still working on something here?

**Comment by koreno** [comment]
> @koreno are you still working on something here?

I have some follow-ups, I'll try to add the little ones at least.

Later I have a big refactor around logging to adapt it to [logbook](https://logbook.readthedocs.io) and new-style formatting...

**Comment by koreno** [comment]
seems like you removed me as a contributor...
you can add me back as korenog@gmail.com

**Pull request closed without merging**